packages/firestore/src/local/indexeddb_mutation_queue.ts

```diff
@@ -352,50 +352,6 @@ export class IndexedDbMutationQueue implements MutationQueue {
       .next(() => results);
   }
 
-  getAllMutationBatchesAffectingDocumentKeys(
```

Comment:
I don't think you can do remove this method as it is used by the LocalDocumentsView and needs to be implemented by both the IndexedDbMutationQueue and the MemoryMutationQueue. You can have a standalone function in this file that you can also call from this method.